impl/pom.xml

```diff
@@ -80,11 +80,11 @@
     </dependency>
     <dependency>
       <groupId>org.hamcrest</groupId>
-      <artifactId>java-hamcrest</artifactId>
+      <artifactId>hamcrest</artifactId>
```

Comment:
You normally don't need Hamcrest anymore, but this repo has a few tests that use some of the more elaborate matchers, so the additional dependency felt justified